ruff.toml

```diff
@@ -0,0 +1,36 @@
+line-length = 80
+include = [
+  "src/*.py",
```

Comment:
shouldn't we just have `**/*.py`, `**/*.ipynb`, `**/*.pyi`? as the pre-commit seems to suggest that's what it's going to look at. Or does the pre-commit use this file when it runs?